docs/software/media.md

```diff
@@ -650,7 +672,7 @@ Turn your device into a lightweight Mumble VoIP server.
 
 === "Murmur Server connection details"
 
-    Use the following credentials to connect clients to the Murmur server:
+    The web interface is accessible via port **64738**:
```

Comment:
That is not a web interface, but indeed only for client connections. Gives an empty response when trying to open in browser. Probably:
```
Clients can connect via port **64738**:
```

Reply:
Ok, done.